stages/org.osbuild.kickstart

```diff
@@ -160,6 +160,15 @@ SCHEMA = r"""
       }
     }
   },
+  "rootpw": {
+    "type": "object",
```

Comment:
What is the longer term idea here, will we provide more of the options outlined in https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#rootpw ? And if so, should "lock" really be required?

Reply:
The plan is indeed to eventually support all the options currently being set in https://github.com/osbuild/images/blob/main/pkg/manifest/anaconda_installer_iso_tree.go#L384 as stage options.